packages/commerce-events-collector-segments/src/index.ts

```diff
@@ -34,4 +41,26 @@ const callProxyService = async (ecid: string) => {
     setAdobeCommerceSegmentCookies(segmentMembershipIds);
 };
 
+const initializeGetAEPSegements = async () => {
+    console.log("initializeGetAEPSegements");
+
+    // let retry = true;
+    const ALLOY_COMMAND = "sendEvent"; //"getIdentity"
+
+    if (window.hasOwnProperty("alloy")) {
```

Comment:
@jasonrpoteet we have a story to make the name of the aep web sdk configurable meaning it won't always be alloy - this code is fine for now, but wouldn't work with a different name

@grahamcrackers can you keep Jason tagged in PRs addressing the web sdk (and data layer) configurable names

Reply:
@jasonrpoteet yes! it's [DINT-636](https://jira.corp.adobe.com/browse/DINT-636)